integration_test/geth_blockchain_test.go

```diff
@@ -20,7 +20,8 @@ var _ = Describe("Reading from the Geth blockchain", func() {
 		inMemory = inmemory.NewInMemory()
 	})
 
-	It("reads two blocks", func(done Done) {
+	// marking this pending because Infura network errors cause flakiness
+	XIt("reads two blocks", func(done Done) {
```

Comment:
I think this is fixed in 9df602d if wanted to pull in here.

Reply:
👍 will rebase once merged